code/modules/clothing/under/miscellaneous.dm

```diff
@@ -11,7 +11,7 @@
 	icon_state = "harness"
 	worn_state = "harness"
 	item_state = "w_suit"
-	species_restricted = null
+	species_restricted = list("Vaurca Worker, Vaurca Warrior, Diona, Industrial Frame, Baseline Frame")

```

Comment:
This has to be a list of strings. Right now it's a list containing one string.

`list("a", "list", "of", "strings")` vs `list("just one string in a list")`